onfido-java/src/test/java/com/onfido/integration/ExtractionManagerTest.java

```diff
@@ -14,15 +14,15 @@
 public class ExtractionManagerTest extends ApiIntegrationTest {
 
     @Test
-    public void createExtraction() throws Exception {
+    public void preformExtraction() throws Exception {
         String response = new JsonObject().add("document_id", "id").toJson();
 
         MockWebServer server = mockRequestResponse(response);
 
         Onfido onfido =
                 Onfido.builder().apiToken("token").unknownApiUrl(server.url("/").toString()).build();
 
-        Extraction extraction = onfido.extraction.create(Extraction.request().documentId("id"));
+        Extraction extraction = onfido.extraction.perform("id");
 
```

Comment:
Can you add a test to make sure the request body is correct? See e.g. the post request for applicants.

Reply:
Add a check for it